.changeset/sour-donuts-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Add possibility to customize actions in CatalogTable
```

Comment:
```suggestion
Adds an optional `actions` prop to `CatalogTable` and `CatalogPage` to support supplying custom actions for each entity row in the table. This uses the default actions if not provided.
```

Since this gets merged into a larger changelog and release notes, better to provide more detail here.